views/admin/events.erb

```diff
@@ -16,17 +16,16 @@
         </div>
     </nav>
     <% end %>
-    <div class="<%= event_div_class %>">
-        <div id="pagetitle">
+    <div class="dcf-tabs-faked <%= event_div_class %>">
```

Comment:
Is there CSS or JS for this `dcf-tabs-faked` class? I couldn't find any.

Reply:
It's just an identifier to so tabs is not generated by javascript (if had class dcf-tabs) and to show that it is not intended to be.

Reply:
The markup used what javascript would have been run.  The way these tabs work is not supported by our DCF tabs javascript.